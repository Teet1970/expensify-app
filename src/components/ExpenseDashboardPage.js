import React from 'react';
import ExpenseList from './expense_list';
import ExpenseListFilters from './ExpenseListFilters';

const ExpenseDashboardPage =() => (
    <div>
     <ExpenseList />
     <ExpenseListFilters />
    </div>
);

export default ExpenseDashboardPage ;