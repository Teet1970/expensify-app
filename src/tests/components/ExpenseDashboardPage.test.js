import React from 'react';
import {shallow} from 'enzyme';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';


test('shoult render ExpenseDashboardPage correctly', ()=> {
    const wrapper = shallow(<ExpenseDashboardPage />);
    expect(wrapper).toMatchSnapshot();
    
});