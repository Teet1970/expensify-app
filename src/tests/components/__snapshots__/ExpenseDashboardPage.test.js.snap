// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shoult render ExpenseDashboardPage correctly 1`] = `
<div>
  <[object Object] />
  <[object Object] />
</div>
`;
